tempesta_fw/t/unit/test_http_parser.c

```diff
@@ -877,6 +877,219 @@ TEST(http_parser, cookie)
 		"\r\n");
 }
 
+TEST(http_parser, req_hop_by_hop)
+{
```

Comment:
Please add test case described above for "Connection: Host\r\n' to mangled message.